controllers/listTestsController.js

```diff
@@ -0,0 +1,36 @@
+import {TestsService} from '../services/testsService';
+const testsService = new TestsService();
+import moment from 'moment';
+
+class ListTestsController {
+	constructor() {
+	}
+
+	get testsService() {
+		return testsService;
+	}
+	showLatestTestSuite(userIdFIlter, callback) {
+		this.testsService.getMostRecentTestSuite(userIdFIlter, (err, mostRecentTestSuite) => {
+			if (!err) {
+				if (mostRecentTestSuite) {
+					var date = new Date();
+					date.setTime(mostRecentTestSuite.timestamp);
+					callback(null, {
+						testSuite: mostRecentTestSuite,
+						formattedTimestamp: moment(date).format('DD/MM/YYYY à HH:mm:ss')
+					});
+				} else {
+					callback(null, {testSuite: null});
+				}
+			} else {
+				callback(err);
+			}
+		});
+	}
+
+	filterMine(filterName, sessionData) {
```

Comment:
Poor naming: this function is not a `filter`, it returns a Boolean and is a filter callback.